### ISSUE TITLE
Change the background color of material website to make it more readable

### ISSUE THREAD

**Issue by jalajc** [angular/material2#9339]
#### feature request, or proposal:
To make the documentation more readable, please change the background color to lighter shade of gray/grey
#### What is the current behavior?
Currently if one navigates to individual components at [material.angular.io/components/](material.angular.io/components/), the Font color and the background color are close in shade. Moreover a thinner (less weight) font make it difficult to read. (to me, this is little against the material design philosophy of readability).

**Comment by willshowell** [comment]
Screenshot?

**Comment by jalajc** [comment]
Here are some... 
![material angular1](https://user-images.githubusercontent.com/13805246/34832562-69faeba0-f710-11e7-9bfe-4b14baa41bcc.png)
![material angular2](https://user-images.githubusercontent.com/13805246/34832563-6b74e9a4-f710-11e7-89d2-f969b04efc2d.png)
![material angular3](https://user-images.githubusercontent.com/13805246/34832805-e218910c-f713-11e7-8816-708aea40454d.png)

**Comment by jelbourn** [comment]
It does like like many of the syntax highlighting colors don't meet W3C contrast standards

**Comment by rafaelss95** [comment]
Also, I find some docs that are difficult to read using the following theme:

![material2-table](https://user-images.githubusercontent.com/11965907/34853918-2ce197d2-f715-11e7-8656-c12290b2104e.png)

![cdk-table](https://user-images.githubusercontent.com/11965907/34853956-6a9e3bb6-f715-11e7-809e-a1f7ac6eef1f.png)

**Comment by jalajc** [comment]
@jelbourn yups, I realize this contrast issue is more than background color. Syntax highlighting too has to improve.and may be font weights too?
Syntax highlighting at angular.io (https://angular.io/tutorial/toh-pt1)seems ok to me though, but don't know if it fits to W3C contrast settings.

**Comment by amcdnl** [comment]
This issue should be logged here: https://github.com/angular/material.angular.io

Closing this and logging over there: https://github.com/angular/material.angular.io/issues/398
